scripts/crontab

```diff
@@ -12,6 +12,7 @@ SHELL=/bin/bash
 0 6 1,16 * * source $HOME/bin/openrc_cpouta_prod_v3.sh; $HOME/diskimage-builder-scripts.cpouta/scripts/image_create_ubuntu-16.04-cuda.sh >> $HOME/log/c/ubuntu-16.04-cuda.log 2>&1
 
 0 7 1,16 * * source $HOME/bin/openrc_cpouta_prod_v3.sh; $HOME/diskimage-builder-scripts.cpouta/scripts/image_create_ubuntu-18.04.sh >> $HOME/log/c/ubuntu-18.04.log 2>&1
+0 7 1,16 * * source $HOME/bin/openrc_cpouta_prod_v3.sh; $HOME/diskimage-builder-scripts.cpouta/scripts/image_create_ubuntu-20.04.sh >> $HOME/log/c/ubuntu-20.04.log 2>&1
```

Comment:
You shouldn't do two builds at the same time, it can screw up the loop mount system

Reply:
Now each imagebuild at different time.

Reply:
```
0 10 * * * [...] >> $HOME/log/c-devel/centos7-cuda.log 2>&1
0 10 1,16 * * [...] >> $HOME/log/c/ubuntu-20.04.log 2>&1
```

These are still overlapping at 10AM on the 1st and 16th day of the month.

I think you need to push one or more of the cpouta-devel builds closer to evening to make this "fit".

Reply:
Yes made another change for ubuntu-20.04. Now this is 30 10 1,6 * *.